web/src/EducationBenchmarking.Web/Views/SchoolPlanningCreate/OtherTeachingPeriods.cshtml

```diff
@@ -0,0 +1,84 @@
+@using EducationBenchmarking.Web.Domain
+@model EducationBenchmarking.Web.ViewModels.SchoolPlanCreateViewModel
+@{
+    ViewData[ViewDataKeys.Title] = PageTitles.SchoolPlanningOtherTeacherPeriodsAdd;
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+            <span class="govuk-caption-l">@Model.Name</span>
+            <h1 class="govuk-fieldset__heading">
+                @ViewData[ViewDataKeys.Title]
+            </h1>
+        </legend>
+        <div class="govuk-inset-text">
+            <p class="govuk-body">
+            These are additional periods that aren't accounted for in the core timetable where a teacher interacts with pupils.
+            For example, intervention, dyslexia support and career support periods.
+        </div>
+
+        <div class="govuk-warning-text">
+            <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+            <strong class="govuk-warning-text__text">
+                <span class="govuk-visually-hidden">Warning</span>
+                You should not include management or PPA time here.
+            </strong>
+        </div>
+    </div>
+</div>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        @using (Html.BeginForm("CreatePlan", "SchoolPlanningCreate", new { urn = Model.Urn, year = Model.Year, step = PlanSteps.OtherTeachingPeriods }, FormMethod.Post, true, new { novalidate = "novalidate" }))
+        {
+            <div class="govuk-form-group">
+                <fieldset class="govuk-fieldset">
+                    <table class="govuk-table">
+                        <thead class="govuk-table__head">
+                        <tr class="govuk-table__row">
+                            <th scope="col" class="govuk-table__header">Name of teaching period</th>
+                            <th scope="col" class="govuk-table__header">Number of periods per timetable cycle</th>
+                            <th scope="col" class="govuk-table__header"></th>
```

Comment:
missing table header which i believe should be "Manage rows"

Reply:
Current system doesn't follow design system